[svelte-check] Cannot find module 'svelte-multiselect' or its corresponding type declarations.

Hello,
I have problems using this library after update to version >= [v8.4.0](https://github.com/janosh/svelte-multiselect/releases/tag/v8.4.0)

```svelte
<script lang="ts">
    import type {Option} from 'svelte-multiselect';
    import MultiSelect from 'svelte-multiselect';

    const opts: Option[] = [1, 2, 3, 4, 5]
</script>


<div class="test">
    This is just a test
</div>
<MultiSelect options={opts}></MultiSelect>
```
When I import from `svelte-multiselect`, svelte-check will throw error: `Cannot find module 'svelte-multiselect' or its corresponding type declarations.`.  Build is ok, code works, only svelte-check is the problem.

I found this svelte/package v2 issue [here](https://github.com/sveltejs/kit/issues/9114).
Did you managed to solve it? Or is it svelte-check issue and I should create issue there?

--- a/package.json
+++ b/package.json
@@ -77,6 +77,7 @@
       "default": "./dist/index.js"
     }
   },
+  "types": "./dist/index.d.ts",
   "files": [
     "dist"
   ]

